spec/stub/formulary.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
All of these stubs are available when run within a real `brew` context, right?

Reply:
Yep.